backends/tfhe-cuda-backend/cuda/src/integer/rerand.cu

```diff
@@ -88,9 +88,6 @@ void cuda_rerand_64(
           " in the interval [256..16384].");
     break;
   }
-
```

Comment:
could you explain why this isn't needed anymore? synchronization is done on the rust side ?

Reply:
I removed sync from release and kept it only in the cleanup function in the end